tests/shell-test/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "eigen-gevulot-test-programs"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+description = "Eigen Gevulot Test Programs"
+documentation = "https://eigen.cash"
+homepage = "https://eigen.cash"
+
+
+[[bin]]
+name = "prover"
+path = "src/prover.rs"
+
+[[bin]]
+name = "verifier"
+path = "src/verifier.rs"
+
+[dependencies]
+starky = { package="starky", path = "../../eigen-zkvm/starky", version = "0.0.1" }
```

Comment:
starky = { git = "https://github.com/0xEigenLabs/eigen-zkvm", branch = "main" }, then you can remove the eigen-zkvm submodule in this repo.